src/common/util/colors.js

```diff
@@ -22,9 +22,15 @@ const interpolateTurbo = (value) => {
   ];
 };
 
-const getSpeedColor = (speed, maxSpeed) => {
-  const normalizedSpeed = Math.max(0, Math.min(1, speed / maxSpeed));
+const getSpeedColor = (speed, minSpeed, maxSpeed) => {
+  if (maxSpeed <= minSpeed) {
+    throw new Error('maxSpeed must be greater than minSpeed');
+  }
```

Comment:
Why do we need this check?

Reply:
We don't. The vars get fed correctly in the step before, and user input does not occur.

I rushed too fast once i spotted the error, this for both of them.